### PR TITLE
Install TDX Cli Script get latest TDX Cli version using Makefile instead of github api

### PR DESCRIPTION
**Pull request by srinics**
These changes introduced because of github api rate limit issue.

### CODE REVIEW

File: release/install-tdx-cli-azure.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ readonly REPO_URL="intel/trustauthority-client-for-go"
 readonly CLI_NAME="Intel Trust Authority Client for Azure"
 readonly RAW_MAKEFILE="https://raw.githubusercontent.com/${REPO_URL}/main/tdx-cli/Makefile"
 if [ -z "${CLI_VERSION}" ]; then
-    CLI_VERSION=$(curl  --silent  https://api.github.com/repos/${REPO_URL}/releases/latest | grep -Po '"tag_name": "\K.*?(?=")')
+    CLI_VERSION=$(curl -s  ${RAW_MAKEFILE} | grep '^VERSION :=' | sed -e "s/\(^VERSION.*\)\(v[0-9]\+.[0-9]\+.[0-9]\+\)/\2/g")
```
Comment (arvind5) [inline release/install-tdx-cli-azure.sh:34]: lets check the exit code of curl command and handle failure case.

File: tdx-cli/trustauthority-cli
Comment (arvind5) [inline tdx-cli/trustauthority-cli:1]: remove binary.

**Review by arvind5 — CHANGES_REQUESTED** [review]

File: release/install-tdx-cli-azure.sh [hunk 2]
```diff
@@ -31,7 +31,13 @@ readonly REPO_URL="intel/trustauthority-client-for-go"
 readonly CLI_NAME="Intel Trust Authority Client for Azure"
 readonly RAW_MAKEFILE="https://raw.githubusercontent.com/${REPO_URL}/main/tdx-cli/Makefile"
 if [ -z "${CLI_VERSION}" ]; then
-    CLI_VERSION=$(curl  --silent  https://api.github.com/repos/${REPO_URL}/releases/latest | grep -Po '"tag_name": "\K.*?(?=")')
+	ret_code=$(curl -s  ${RAW_MAKEFILE} -w '%{http_code}' -o /tmp/cli_version)
```
Comment (arvind5) [inline release/install-tdx-cli-azure.sh:34]: keep the existing indentation.

**Review by arvind5 — CHANGES_REQUESTED** [review]

File: release/install-tdx-cli-azure.sh (hunk 2 above)
Comment (srinics) [reply release/install-tdx-cli-azure.sh]: Addressed

**Review by arvind5 — APPROVED** [review]

**Pull request merged**